app/components/print-this.js

```diff
@@ -1,8 +1,26 @@
 import Ember from 'ember';
+
+const {
+  get,
+  isEmpty
+} = Ember;
+
 export default Ember.Component.extend({
+  print: null,
```

Comment:
print should default to true; otherwise invoice printing won't work unless you change that template.  I would prefer to change the default to true.